test/hotspot/jtreg/compiler/intrinsics/bmi/verifycode/TZcntTestI.java

```diff
@@ -50,6 +50,10 @@ protected TZcntTestI(Method method) {
 
         instrMask_x64 = new byte[]{(byte) 0xFF, (byte) 0x00, (byte) 0xFF, (byte) 0xFF};
         instrPattern_x64 = new byte[]{(byte) 0xF3, (byte) 0x00, (byte) 0x0F, (byte) 0xBC};
+
+        // REX2 variant
+        instrMaskAPX = new byte[]{(byte) 0xFF, (byte)0x80, (byte) 0xFF};
+        instrPatternAPX = new byte[]{(byte) 0xD5, (byte) 0x80, (byte) 0xBC};
```

Comment:
I think we should check for 0xF3 as well here for tzcnt to differentiate it from bsf:
       instrMaskAPX = new byte[]{(byte) 0xFF, (byte) 0xFF,  (byte)0x80, (byte) 0xFF};
       instrPatternAPX = new byte[]{(byte) 0xF3, (byte) 0xD5, (byte) 0x80, (byte) 0xBC};